django/db/models/options.py

```diff
@@ -545,6 +545,8 @@ def get_field(self, field_name):
         """
         Return a field instance given the name of a forward or reverse field.
         """
+        if field_name == 'pk':
+            return self.pk
```

Comment:
Circling back to these comments some of the things I said don't seem accurate. I'm currently in the midst of going through the life cycle of `Model`s and `Option`s and will report back on what I find.